data/json/mutations/mutations.json

```
@@ -1664,8 +1786,8 @@
     "name": { "str": "Cephalopod Vision" },
     "points": 3,
     "description": "Your brain has caught up with your eyes.  You can see much better in the dark, but sunlight seems much brighter now.  Activate to toggle NV-visible areas on or off.",
+    "types": [ "NIGHTVISION" ],
```

```suggestion
    "types": [ "VISION" ],
```
It will also be EYES when it grants replacement eyes instead of being an on-top trait, but that's for #53004